src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/modelchecking/ModelCheckingStrategy.kt

```diff
@@ -310,4 +454,12 @@ internal class ModelCheckingStrategy(
 
         fun build() = Interleaving(switchPositions, threadSwitchChoices, lastNoninitializedNode)
     }
+
+    companion object {
+        /**
+         * We provide lincheck version to [testFailed] method to the plugin be able to
+         * determine if this version is compatible with the plugin version.
+         */
+        private val lincheckVersion = System.getProperty("lincheck.version")
```

Comment:
Users will not pass the Lincheck version this way. Please find another solution.

Reply:
Solved